site/content/docs/4.3/components/offcanvas.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: docs
+title: Offcanvas
+description: Toggle the visibility sidebars across your project with a few classes and our JavaScript plugin.
+group: components
+toc: true
+---
+
+## How it works
+
+The offcanvas JavaScript plugin is used to show and hide a sidebar (or bottom-bar). Buttons or anchors are used as triggers, to specific elements you toggle.
+
+Given how CSS handles animations, you cannot use `margin` & `translate` on a `.offcanvas` element. Use the class on an independent wrapping element.
+
+## Example
+
+Click the buttons below to show and hide another element via class changes:
+
+- `.offcanvas` hides content
+- `.offcanvas.show` shows content
+
+You can use a link with the `href` attribute, or a button with the `data-target` attribute. In both cases, the `data-toggle="offcanvas"` is required.
+
+{{< example >}}
+<p>
+  <a class="btn btn-primary" data-toggle="offcanvas" href="#offcanvasExample" role="button" aria-expanded="false" aria-controls="offcanvasExample">
+    Link with href
+  </a>
+  <button class="btn btn-primary" type="button" data-toggle="offcanvas" data-target="#offcanvasExample" aria-expanded="false" aria-controls="offcanvasExample">
+    Button with data-target
+  </button>
+</p>
+<div class="offcanvas offanvas-push-body bg-dark text-white" tabindex="-1" id="offcanvasExample">
+  <div class="offcanvas-header">
+    <div class="h4 col text-center">Offcanvas</div>
+    <button type="button" class="close text-reset" data-dismiss="offcanvas" aria-label="Close">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="card card-body text-dark ">
+    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident.
+  </div>
+
+  <div class="dropdown mt-3">
+    <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-expanded="false">
+      Dropdown button
+    </button>
+    <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+      <li><a class="dropdown-item" href="#">Action</a></li>
+      <li><a class="dropdown-item" href="#">Another action</a></li>
+      <li><a class="dropdown-item" href="#">Something else here</a></li>
+    </ul>
+  </div>
+</div>
+{{< /example >}}
+
+## Position Modifiers
```

Comment:
We use sentence case in Bootstrap, so this would be `Position modifiers`, but I think we can leave just as `Position` for brevity's sake.

```suggestion
## Position
```